security/rbac/rbac-docker/README.md

```diff
@@ -2,7 +2,6 @@
   
 # Overview
 
-These demos showcase the Role Based Access Control (RBAC) functionality in Confluent Platform. It is mostly for reference to see a workflow using the new RBAC feature across the services in Confluent Platform.
-
-The documentation for running this demo is at [https://docs.confluent.io/current/tutorials/examples/security/rbac/docs/index.html](https://docs.confluent.io/current/tutorials/examples/security/rbac/docs/index.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.rbac)
+The Docker-based Role Based Access Control (RBAC) demo is deprecated.
 
+Instead, see [cp-demo](https://docs.confluent.io/current/tutorials/cp-demo/docs/index.html?utm_source=github&utm_medium=demo&utm_campaign=ch.cp-demo_type.community_content.rbac), which is more representative of a real deployment of an Apache Kafka® event streaming application, with RBAC and other Confluent Platform security features and LDAP integration.
```

Comment:
should probably trim out the tracking portion of that URL, e.g. 

```
Instead, see [cp-demo](https://docs.confluent.io/current/tutorials/cp-demo/docs/index.html)...
```

Reply:
@joel-hamill It's for tracking between GH and docs (this is in the README)